_posts/2025-01-11-week8.markdown

```diff
@@ -0,0 +1,13 @@
+
+#### Takeaway:
+-While my dad guided me through the cleanup process, I tried to impress him by citing a coding philosophy I heard which stated "if it works don't fix it." He wasn't impressed. This philosphy didn't necessarily apply anyway, seeing as my code was hardly functional. But his point was that messy code had no longevity. He said the creater of Friendster (a precursor to Facebook if you didn't know) should be one of the richest men on earth, but that the app crumbled when its user base skyrocketed. The program couldn't keep up, and when it came time to expand, they instead drowned in unfixable, unreadable code. I will continue to start messy, but I am going to try to make my code easy to reorganize. Because truly, **readability is the key to longterm success**.
```

Comment:
Good takeaway, but I'd say "readability is one of the most important things in code".  When I hear "long term success", I think of the overall project/company, and there are 1000 things involved with that.